api/swagger/swagger.yaml

```diff
@@ -294,6 +294,23 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
 
+  /sources:
+    x-swagger-router-controller: sources
+    get:
+      tags:
+        - sources
+      description: Returns list of sources
+      operationId: list
```

Comment:
I was going to add a comment on this change. I like using general names like `list` or `sources` because when you're using the `swagger.yaml` in a client you can do `client.sources.list()` instead of `client.sources.getSources()`. This makes it more succinct , as in `client.persons.get(1)` and `client.organisations.get(1)` instead of `client.persons.getPerson(1)` and `client.organisations.getOrganisation(1)`.